examples/http-client-endpoint/http_client_endpoint.bal

```diff
@@ -1,8 +1,19 @@
 import ballerina/http;
 import ballerina/io;
 
+// To secure an endpoint using HTTPS, the endpoint needs to be configured with
+// a truststore for the endpoint.
+http:ClientEndpointConfig clientConf = {
+    secureSocket:{
+        trustStore:{
+            path: "${ballerina.home}/bre/security/ballerinaTruststore.p12",
+            password: "ballerina"
+        }
+    }
+};
+
 // Creates a new client with the backend URL.
-http:Client clientEndpoint = new("https://postman-echo.com");
+http:Client clientEndpoint = new("https://postman-echo.com", clientConf);
```

Comment:
Guys, can we use some http endpoint. Like below,

http://api.sunrise-sunset.org